route/service.go

```diff
@@ -0,0 +1,55 @@
+package route
+
+import (
+	"fmt"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/thunderdb/ThunderDB/consistent"
+	"github.com/thunderdb/ThunderDB/proto"
+)
+
+type DhtService struct {
+	thisNode proto.Node
+	hashRing *consistent.Consistent
+}
+
+func NewDhtService() *DhtService {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function NewDhtService should have comment or be unexported